poll-webapp/src/main/webapp/skin/less/poll.less

```diff
@@ -123,14 +128,14 @@
           left: 0;
           bottom: 0;
           z-index: 0;
-          background-color: @greyColor;
+          background-color: @greyColorDefault;
           border-top-left-radius: 5px;
           border-bottom-left-radius: 5px;
           transition: all .3s cubic-bezier(0.5,1.2,.5,1.2);
           -webkit-transition: all .3s cubic-bezier(0.5,1.2,.5,1.2);
           -moz-transition: all .3s cubic-bezier(0.5,1.2,.5,1.2);
           &.selected {
-            background-color: @primaryColor;
+            background-color: #C7D2E1;
```

Comment:
the same remark as above, we can not use color values here too

Reply:
Hello. Here we want to remove the usage of primary color. It is ask by the task. 
I'm not sure there is an existing variable matching this color value.

Reply:
this code is not declared  in files variables.less